### ISSUE TITLE
Variables in Action names stopped working when they are in conditional [ ] brackets

### ISSUE THREAD

**Issue by lmagyar** [nielsfaber/scheduler-card#591]
### Checklist

- [X] I checked for similar existing issues (both open and closed) before posting.
- [X] I will participate in further discussion about this issue and can help by testing (if requested).

### Card Version

v2.3.5 (currently latest)

### Component Version

v3.2.10 (currently latest)

### Bug description

Default and custom action names containing variables stopped displaying the variable value.

Eg. default config: `"cool[ to {temperature}]"` won't display the temperature, only `Cool`.

If I manually configure an action, like:
```
                      actions:
                        - service: set_temperature
                          service_data:
                            hvac_mode: cool
                          name: "coolx[ to {temperature}]x"
                          icon: "mdi:snowflake"
                          variables:
                            temperature:
                              name: temperature
                              min: 20
                              max: 30
                              step: 0.5
                              unit: "°C"
```

It displays only `Coolxx`

---

BUT!!!

If I remove the conditional `[]` like `name: "cool to {temperature}"`, it works, displays `Cool to 28°C`, so the `{temperature}` variable has value, only the evaluation of the conditional `[]` got broken lately.

### Steps to reproduce

Try to display any schedule that has variable in it's action name.

Even a vanilla config is enough, like:

```
type: custom:scheduler-card
```

**Comment by nielsfaber** [comment]
Well spotted, I had no idea this was broken!
It should be fixed in the v3.2.6 release.

**Comment by lmagyar** [comment]
Wow, that was fast! :) I confirm, this works again. :)
